audio/portaudio/Portfile

```diff
@@ -26,35 +25,26 @@ master_sites        ${homepage}archives
 distname            pa_stable_v[strsed ${version} {g/\./_/}]
 extract.suffix      .tgz
 
-checksums           rmd160  0a290cea8a102a45b2df9b72046b3eed0ab35b4f \
-                    sha256  8fe024a5f0681e112c6979808f684c3516061cc51d3acc0b726af98fc96c8d57
+checksums           rmd160  e6e5cd3f3cb7469aa17549c189e445d573567e13 \
+                    sha256  f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
 
 worksrcdir          ${name}
 
 # By default, PortAudio is built as a universal binary.
 # To build a non-universal library for the host architecture,
 # simply use the --disable-mac-universal option with configure.
 # http://www.portaudio.com/trac/wiki/TutorialDir/Compile/MacintoshCoreAudio
-configure.args			\
-    --enable-option-checking	\
-    --enable-cxx		\
-    --disable-mac-universal	\
-    --without-alsa		\
-    --without-jack		\
-    --without-oss		\
-    --without-asihpi		\
-    --without-winapi
-
```

Comment:
I'm confused by this change. We don't have jack or alsa or oss on MacOS so why would you want to enable them? Also, why do you want to build universal binaries?

Reply:
Maybe in many cases it'd be fine without them but chiefly because many people (myself included) use mac to write portable code that will be used in other platforms as well. And since the whole appeal of portaudio is in it being a portable audio library, I think it's preferable to have all components by default, rather than the opposite. Of course, we could sort it out with variants as well I guess.

Reply:
Also, I _don't_ want universal binaries. In fact, my portfile (and the original as well) explicitly disables them?

Reply:
You can't build those components on a mac, and there's no reason you need them in order to develop code.

Reply:
Sorry about the misunderstanding about the disable line, though.

Reply:
I guess you're right, as I said I made this a while ago and for some things I was sort-of fumbling around to see what worked. I'll correct this.